lblod/frontend-form-builder
# Add more validations per field type

## ID
 [SFB-39](https://binnenland.atlassian.net/browse/SFB-39)

 ## Description

The `form-builder`  now only has two possible validations. Those are the `RequiredConstraint` & `Maxlength` as there are much more validations than those two I am extending the list of validations that are possible for the different `field types` in this PR. 

In the [ticket above](https://binnenland.atlassian.net/browse/SFB-39) you will find a list of all the field types with the validations I have added in this PR.

 ## Type of change


 ## How to test

Add two or three fields to your form with a different `display-type`. Switch to the validations tab an you will see that the options per `display-type` are different. Try out all the validations that you can find in the dropdown.

Try to break it so our users will not stumble on this error.

 ## Links to other PR's

- /
